public_html/wp-content/plugins/wordcamp-remote-css/app/synchronize-remote-css.php

```diff
@@ -76,7 +76,14 @@ function fetch_unsafe_remote_css( $remote_css_url ) {
  * @throws Exception
  */
 function sanitize_unsafe_css( $unsafe_css ) {
-	require_once JETPACK__PLUGIN_DIR . '/modules/custom-css/custom-css-4.7.php';
+	// Jetpack < 11.6 support.
+	$jetpack_custom_css_file = JETPACK__PLUGIN_DIR . '/modules/custom-css/custom-css-4.7.php';
+	if ( ! file_exists( $file ) ) {
```

Comment:
```suggestion
	if ( ! file_exists( $jetpack_custom_css_file ) ) {
```

It doesn't look like `$file` exists, I'm assuming you meant `$jetpack_custom_css_file`